### PR TITLE
add zga 0.0.7

### PR DESCRIPTION
**Pull request by laxeye**
New recipe for zga - https://github.com/laxeye/zga - prokaryotic genome assembly and annotation pipeline written in python.

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by laxeye** [comment]
@BiocondaBot please add label

File: recipes/zga/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  imports:
+    - zga
+  commands:
+    - zga --help
+
+about:
+  home: "https://github.com/laxeye/zga"
+  license: BSD
+  license_family: BSD
+  license_file:
```
Comment (mbhall88) [inline recipes/zga/meta.yaml:50]:
```suggestion
```

File: recipes/zga/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  commands:
+    - zga --help
+
+about:
+  home: "https://github.com/laxeye/zga"
+  license: BSD
+  license_family: BSD
+  license_file: 
+  summary: "Prokaryotic genome assembly and annotation pipeline"
+  doc_url:
```
Comment (mbhall88) [inline recipes/zga/meta.yaml:52]:
```suggestion
```

**Comment by laxeye** [comment]
@mbhall88 Thank You for the suggested edits!

**Comment by mbhall88** [comment]
@bioconda-bot please merge

**Pull request merged**